services/openvpn/session/validator_mocks.go

```diff
@@ -22,10 +22,21 @@ import (
 	"github.com/mysteriumnetwork/node/session"
 )
 
-const mockedVPNConfig = "config_string"
+func mockStorage(sessions ...session.Session) *session.Storage {
```

Comment:
Doesn't look like mock storage to me, but a real prepopulated storage. Again - missing Storage interface ?

Reply:
Mocked storage